src/script/Config.ts

```diff
@@ -22,14 +22,13 @@ import {createRandomUuid} from 'Util/util';
 import {Runtime} from '@wireapp/commons';
 const env = window.wire.env;
 
-export const ACCENT_ID = {
+export const ACCENT_ID: {[key: string]: number} = {
```

Comment:
```suggestion
export const ACCENT_ID: Record<string, number> = {
```
this feels more readable to me

Reply:
👍

Reply:
resolved